packages/netlify-cms-widget-markdown/src/serializers/remarkSlate.js

```diff
@@ -1,31 +1,4 @@
-import { isEmpty, isArray, last, flatMap } from 'lodash';
```

Comment:
The heart of our markdown editor overhaul, together with slateRemark.js, this module parses MDAST to a Slate document.